packages/web3-utils/src/big_decimal.ts

```diff
@@ -0,0 +1,51 @@
+export class BigDecimal {
+	private readonly bigint: bigint;
+	private readonly decimals: number;
+
+	public constructor(value: bigint | string, decimals?: number) {
```

Comment:
Couldn't `value` be of type `Numbers`?

Reply:
No as per the logic for divide/multiply. Value can be either `bigint` and `decimal` or just `string` value.